lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -64,7 +66,18 @@ import Servant.Client.Core
 import Servant.Links
     ( Link, safeLink )
 
--- TODO: Implement a NetworkLayer
+mkNetworkLayer :: MonadIO m => JormungandrLayer m -> NetworkLayer t m
+mkNetworkLayer j = NetworkLayer
+    { networkTip = do
+        t@(BlockId hash) <- lift $ getTipId j
```

Comment:
Thinking `BlockId` might be a good candidate for a `type`alias.

Reply:
Rule of thumb: nothing that short is a good idea for a type-alias :smile:

Reply:
:(

impossible regardless https://github.com/input-output-hk/cardano-wallet/pull/383#discussion_r291573606